packages/suspense/package.json

```diff
@@ -67,10 +67,10 @@
     "point-utilities": "^0.0.2"
   },
   "peerDependencies": {
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0"
```

Comment:
I'm not sure what the best way is to support both, so for now I'm only focused on getting it to work with 19.